org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.xml

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
 <node>
   <interface name="org.opent2t.sample.binaryswitch.superpopular">
+    <description>OpenT2T SuperPopular Binary Switch</description>
+
```

Comment:
Super Nit: There seems to be an inconsistency wrt newlines in the .xml definitions  - here for eg vs lamp.superpopular.xml -- are we trying t obe consistent any which way?